sortable-list.html

```diff
@@ -101,44 +101,54 @@
         evt.dataTransfer.effectAllowed = "move";
         evt.dataTransfer.setData("text", index+','+sortableList.group);
         this.classList.add('dragged');
-        // EDGE does not support setDragImage natively
-        if (sortableList.isEdge || sortableList.isIE) {
-          sortableList.setDragImageEdge(this, sortableList, evt.x, evt.y);
+
+        if (sortableList.isEdge || sortableList.isIE || sortableList.isFF) {
+          sortableList.ghostEl = sortableList.buildGhost(this);
+          if (sortableList.isFF) {
+            // FF does support setDragImage but drag image does not render anything in a shadowRoot
+            sortableList.appendChild(sortableList.ghostEl);
+            sortableList._setGhostOffscreen(sortableList.ghostEl);
+            evt.dataTransfer.setDragImage(sortableList.ghostEl, 0, 0);
+          } else {
+            // EDGE and IE does not support setDragImage natively
+            // make ghostEl follow the mouse around to create the illusion of dragImage
+            // in order for that to happen, ghostEl must be appended to document.body
+            document.body.appendChild(sortableList.ghostEl);
+            sortableList.setDragImageEdge(this, sortableList, evt.x, evt.y);
```

Comment:
lets encapsulate the interior of this if block to keep the `dragStart` logic cleaner.